### ISSUE TITLE
Integrate some of bqt

### ISSUE THREAD

**Issue by huntfx** [huntfx/vfxwindow#5]
A coworker came across [this](https://github.com/techartorg/bqt) for Blender, will have a look at some point to see if it's worth integrating any of it.

**Comment by hannesdelbeke** [comment]
hey recently appointed maintainer of bqt here
~~current version of bqt has some bugs but we have PRs up that address this~~
edit: PR is merged in now

**Comment by huntfx** [comment]
Ah nice one, and thanks for the reminder too, I still need to set aside a bit of time to properly look into this :)

**Comment by hannesdelbeke** [comment]
think the easiest thing would be to just steal the stylesheet from bqt
there also is one for unreal on github you could steal

**Comment by hannesdelbeke** [comment]
https://github.com/leixingyu/UnrealStylesheet

**Comment by huntfx** [comment]
Thanks a bunch, that's be super helpful :)   It currently only uses `QPalette` for the colours, so I'll just need to code in a bit for stylesheets too (or see if I can use those to generate a palette).

With bqt I was mainly hoping I might be able to get a better integration with Blender, similar to how I've got Maya and Nuke set up. We very rarely use Blender at work though so I haven't had much chance to experiment with it yet.

**Comment by hannesdelbeke** [comment]
yes bqt has great integration, ex. widgets stay in foreground etc. it wraps blender in a PySide2 qtWindow. (i've gotten it to work with Qt5.py in the past, think there were some issues with Qt.py)

it does rely on wrapping blender on startup, so it's not a widget you can just inherit.
where as it seems vfxwindow is just a widget class? (forgive my ignorance if wrong)
so ideally the user would install bqt, and then use the vfxwindow widget is how i imagine it
